lib/App/cpanminus/script.pm

```diff
@@ -869,7 +869,7 @@ Options:
   -n,--notest               Do not run unit tests
   --test-only               Run tests only, do not install
   -S,--sudo                 sudo to run install commands
-  --installdeps             Only install dependencies
+  --installdeps             Only install 'test' dependencies
```

Comment:
It's the wording from the in-code comment at https://github.com/miyagawa/cpanminus/blob/devel/lib/App/cpanminus/script.pm#L2212